packages/react/src/components/Tree/TreeItem.tsx

```diff
@@ -78,20 +91,16 @@ class TreeItem extends AutoControlledComponent<ReactProps<TreeItemProps>, TreeIt
     accessibility: defaultBehavior,
   }
 
-  handleTitleOverrides = predefinedProps => ({
+  handleTitleOverrides = (predefinedProps: TreeItemProps) => ({
```

Comment:
Isn't the `predefinedProps` of type `TreeTitleProps`? @layershifter

Reply:
Yep, my typo, great catch 👍